[2022-07-05 14:56:07,646] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: ETL_ANP_Raízen.get_file manual__2022-07-05T14:56:02.447491+00:00 [queued]>
[2022-07-05 14:56:07,673] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: ETL_ANP_Raízen.get_file manual__2022-07-05T14:56:02.447491+00:00 [queued]>
[2022-07-05 14:56:07,673] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-07-05 14:56:07,673] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-07-05 14:56:07,674] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-07-05 14:56:07,720] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): get_file> on 2022-07-05 14:56:02.447491+00:00
[2022-07-05 14:56:07,731] {standard_task_runner.py:52} INFO - Started process 514 to run task
[2022-07-05 14:56:07,742] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'ETL_ANP_Raízen', 'get_file', 'manual__2022-07-05T14:56:02.447491+00:00', '--job-id', '69', '--raw', '--subdir', 'DAGS_FOLDER/ETL_Raizen.py', '--cfg-path', '/tmp/tmpqz0l171v', '--error-file', '/tmp/tmpb0xu_bdk']
[2022-07-05 14:56:07,744] {standard_task_runner.py:80} INFO - Job 69: Subtask get_file
[2022-07-05 14:56:07,910] {task_command.py:369} INFO - Running <TaskInstance: ETL_ANP_Raízen.get_file manual__2022-07-05T14:56:02.447491+00:00 [running]> on host e795889a6d4b
[2022-07-05 14:56:08,495] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=ETL_ANP_Raízen
AIRFLOW_CTX_TASK_ID=get_file
AIRFLOW_CTX_EXECUTION_DATE=2022-07-05T14:56:02.447491+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-07-05T14:56:02.447491+00:00
[2022-07-05 14:56:10,715] {python.py:173} INFO - Done. Returned value was: None
[2022-07-05 14:56:12,220] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=ETL_ANP_Raízen, task_id=get_file, execution_date=20220705T145602, start_date=20220705T145607, end_date=20220705T145612
[2022-07-05 14:56:12,567] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-05 14:56:13,327] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
